src/pages/journal/journal.ts

```diff
@@ -1,11 +1,422 @@
 import { Component } from '@angular/core';
 import { IonicPage, NavController, NavParams } from 'ionic-angular';
+import { ExaminerWorkSchedule } from '../../common/domain/Journal';
 
 @IonicPage()
 @Component({
   selector: 'page-journal',
   templateUrl: 'journal.html'
 })
 export class JournalPage {
-  constructor(public navCtrl: NavController, public navParams: NavParams) {}
+  private journalJson: ExaminerWorkSchedule; // this should be populated by a call to a service
+  public journalJsonText: string; 
+  public journalSlot: any; 
+
+  constructor(public navCtrl: NavController, public navParams: NavParams) {
+    this.journalJson= this.getMockJournal();
```

Comment:
Lint issue

Reply:
Pre-push hook should have prevented this, is it running for you? If not, our linting rules need some tightening.

Reply:
Okay, will create a ticket to roll out an improved set of linting rules across the board.